swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, age=None, condition=None, category=None):
+        self.age = age if age is not None else 0
+        self.condition = condition if condition is not None else 0.0
+        category = category if category is not None else ""
+        self.category = category
+    def __str__(self):
+        return f"Hello World!"
+    def condition_description(self):
+        if 0 <= self.condition < 1: 
+            return f"Heavily used"
+        elif 1 <= self.condition < 2:
+            return f"Used!"
+        elif 2 <= self.condition < 3:
+            return f"Good!"
+        elif 3 <= self.condition < 4:
+            return f"New!"
+        elif 4 <= self.condition <= 5:
+            return f"Mint!"
```

Comment:
Due to the explicit logic checks, we would need to add additional test cases to ensure that each of the possible descriptions is returned for each possible condition, since each condition could potentially have a bug in it.

When we have a long list of chained checks that are very similar, we could try coming up with a way to represent it with data and use iteration. A basic example of this that would mirror your existing behavior could look like:
```python
# a constant variable outside of the class definition
DESCRIPTIONS = ( 
"Heavily used", 
"Used!", 
"Good!", 
"New!", 
"Mint!", 
)

# in Item class definition 
def condition_description(self): 
    return DESCRIPTIONS[self.condition]
```

There are still edge case behaviors we'd need to consider and handle: what if the condition is outside the expected range? what if the condition is a non-integer value?